<?xml version="1.0" encoding="utf-8"?>

<robot name="vss_robot" xmlns:xacro="http://www.ros.org/wiki/xacro">

  <xacro:arg name="robot_number" default="0" />
  <xacro:arg name="is_yellow" default="0" />

  <xacro:property name="robot_number" value="$(arg robot_number)" />
  <xacro:property name="is_yellow" value="$(arg is_yellow)" />

  <!-- Import all Gazebo-customization elements, except colors -->
  <xacro:include filename="$(find travesim)/urdf/generic_vss_robot.gazebo" />
  <xacro:include filename="$(find travesim)/urdf/generic_vss_robot.color" />

  <!-- Code exported from SolidWorks -->
  <xacro:include filename="$(find travesim)/urdf/generic_vss_robot.urdf" />

  <!-- Definition of motor -->
  <xacro:include filename="$(find travesim)/urdf/motor.xacro"/>

  <xacro:motor side="esq" gear_rate="1"/>
  <xacro:motor side="dir" gear_rate="1"/>

</robot>
